packages/notification-center/src/index.test.ts

```diff
@@ -0,0 +1,217 @@
+import {
+  IStoreQuery,
+  IUserPreferenceSettings,
+  IPreferenceChannels,
+  ScreensEnum,
+  ColorScheme,
+  IMessage,
+  ChannelTypeEnum,
+  IMessageCTA,
+  IActor,
+  ActorTypeEnum,
+  ChannelCTATypeEnum,
+  IMessageAction,
+  MessageActionStatusEnum,
+  IMessageButton,
+  ButtonTypeEnum,
+} from './index';
+
+describe('@novu/notification-center - general export interface', () => {
+  it('validate IStoreQuery interface', () => {
+    const storeQuery: IStoreQuery = {
+      feedIdentifier: 'abc123',
+      seen: true,
+      read: true,
+    };
```

Comment:
I am not sure if it is the best way to do the test. what I want to test here is what the package export.